advanced-integration-legacy/README.md

```diff
@@ -0,0 +1,9 @@
+# Advanced Integration Example
+
+## Instructions
+
+1. Rename `.env.example` to `.env` and update `CLIENT_ID` and `APP_SECRET`.
```

Comment:
I don't think there is an `.env.example` in this directory (advanced-integration-legacy)